packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1036,6 +1037,64 @@ module.exports = function (log, config, bounces, statsd) {
     });
   };
 
+  Mailer.prototype.verifyAccountChangeEmail = async function (message) {
+    log.trace('mailer.verifyAccountChangeEmail', {
+      email: message.email,
+      uid: message.uid,
+    });
+
+    const templateName = 'verifyAccountChange';
+    const query = {
+      code: message.code,
+      uid: message.uid,
+    };
+
+    if (message.service) {
+      query.service = message.service;
+    }
+    if (message.redirectTo) {
+      query.redirectTo = message.redirectTo;
+    }
+    if (message.resume) {
+      query.resume = message.resume;
+    }
+
+    const links = this._generateLinks(
+      this.verifyLoginUrl,
+      message,
+      query,
+      templateName
+    );
+    const [time, date] = this._constructLocalTimeString(
+      message.timeZone,
+      message.acceptLanguage
+    );
+
+    const headers = {
+      'X-Signin-Verify-Code': message.code,
```

Comment:
Can we change this `X-Account-Change-Verify-Code`? This will help disambiguate the emails we are looking for in our tests.